api/src/main/java/marquez/logging/LoggingMdcFilter.java

```diff
@@ -28,19 +32,30 @@ public class LoggingMdcFilter
   private static final String REQUEST_ID = "requestID";
   private static final String METHOD = "method";
   private static final String PATH = "path";
+  private static final String PATH_WITH_PARAMETERS = "pathWithParams";
+
+  @Context private ExtendedUriInfo uriInfo;
 
   @Override
   public void onComplete(Throwable throwable) {
     MDC.remove(REQUEST_ID);
     MDC.remove(METHOD);
     MDC.remove(PATH);
+    MDC.remove(PATH_WITH_PARAMETERS);
   }
 
   @Override
   public void filter(ContainerRequestContext requestContext) throws IOException {
     MDC.put(REQUEST_ID, UUID.randomUUID().toString());
     MDC.put(METHOD, requestContext.getMethod());
     MDC.put(PATH, requestContext.getUriInfo().getPath());
+    StringBuilder pathWithPlaceholders = new StringBuilder();
+
+    List<UriTemplate> matchedTemplates = uriInfo.getMatchedTemplates();
+    for (int i = matchedTemplates.size() - 1; i >= 0; i--) {
+      pathWithPlaceholders.append(matchedTemplates.get(i).getTemplate());
+    }
+    MDC.put(PATH_WITH_PARAMETERS, pathWithPlaceholders.toString());
```

Comment:
Can we have a test which that calls some Marquez API endpoint and then calls metrics endpoint to see if the metrics with desired name are present?

Reply:
Added test for that.